include/astra_camera/ros12_shim.h

```diff
@@ -1,24 +1,15 @@
 #ifndef ROS12_SHIM_H_
 #define ROS12_SHIM_H_
 
-#include <stdio.h>
 #include <iostream>
 
-#ifndef ROS_INFO
-  #define ROS_INFO(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
-#ifndef ROS_DEBUG
-  #define ROS_DEBUG(str, ...)
-#endif
-#ifndef ROS_WARN
-  #define ROS_WARN(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
-#ifndef ROS_ERROR
-  #define ROS_ERROR(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
-#ifndef ROS_FATAL
-  #define ROS_FATAL(str, ...) printf(str "\n", ## __VA_ARGS__)
-#endif
+#include <rcutils/logging_macros.h>
+
+#define ROS_INFO RCUTILS_LOG_INFO
+#define ROS_DEBUG RCUTILS_LOG_DEBUG
+#define ROS_WARN RCUTILS_LOG_WARN
+#define ROS_ERROR RCUTILS_LOG_ERROR
+#define ROS_FATAL RCUTILS_LOG_FATAL
```

Comment:
I am unfamiliar with how this was used for shiming, it does look like this file was ROS2 agnostic so far. 
Maybe we should keep the ifndef and include ros2 macros headers only if ROS1 macros not already defined?

Reply:
This file is really meant to be used to provide ROS1->ROS2 porting macros and other niceties.  I don't think we really need the `#ifndef` sections anymore; we are definitely never including any ROS1 headers in this codebase, so they will never be defined by something else.

Reply:
Fair enough, let's remove all of them from this file then

Reply:
Good idea.  Done in 0f2237f.